spec/system/documents/archive_spec.rb

```diff
@@ -61,9 +61,8 @@
       click_link "Archive document"
     end
 
-    it "Archive page contains radio buttons and image" do
-      expect(page).to have_text "Missing scale bar or north arrow"
-      expect(page).to have_css(".govuk-radios__item")
+    it "Archive page contains expected text" do
+      expect(page).to have_text "Why do you want to archive this document?"
```

Comment:
Not sure this is really testing anything here, and we can probably just delete it?